test/integration/targets/callback_default/runme.sh

```diff
@@ -15,6 +15,8 @@ set -eux
 
 run_test() {
 	local testname=$1
+	local TEST_FOLDER_PATH
```

Comment:
There's no need to do this. You can simple replace `$ANSIBLE_PLAYBOOK_DIR` with `TEST_FOLDER_PATH`.
```suggestion

```

Reply:
ANSIBLE_PLAYBOOK_DIR does not seem to resolve correctly in the test script. It looks like it is not set

Reply:
`ANSIBLE_PLAYBOOK_DIR` is set by `ansible-test`, so it will be defined. Here's some of the test output showing when it is set:

```
# ansible-test integration --docker centos7 callback_default -vvv
...
Running callback_default integration test script
Preparing temporary directory: /root/ansible/test/results/.tmp/integration/callback_default-HaGZIh-ÅÑŚÌβŁÈ
Copying test/integration/targets/callback_default/ to /root/ansible/test/results/.tmp/integration/callback_default-HaGZIh-ÅÑŚÌβŁÈ/test/integration/targets/callback_default/
Copying /root/ansible/test/lib/ansible_test/_data/ansible.cfg to /root/ansible/test/results/.tmp/integration/callback_default-HaGZIh-ÅÑŚÌβŁÈ/test/integration/integration.cfg
Copying /root/ansible/test/lib/ansible_test/_data/inventory to /root/ansible/test/results/.tmp/integration/callback_default-HaGZIh-ÅÑŚÌβŁÈ/test/integration/inventory
Injecting "/tmp/python-XzS8nf-ansible/python" as a symlink to the "/usr/bin/python2.7" interpreter.
Run command: ./runme.sh -vvv
Working directory: /root/ansible/test/results/.tmp/integration/callback_default-HaGZIh-ÅÑŚÌβŁÈ/test/integration/targets/callback_default
Program found: ./runme.sh
ANSIBLE_CALLBACK_WHITELIST=junit
ANSIBLE_CONFIG=/root/ansible/test/results/.tmp/integration/callback_default-HaGZIh-ÅÑŚÌβŁÈ/test/integration/integration.cfg
ANSIBLE_DEPRECATION_WARNINGS=false
ANSIBLE_FORCE_COLOR=true
ANSIBLE_FORCE_HANDLERS=true
ANSIBLE_HOST_KEY_CHECKING=false
ANSIBLE_HOST_PATTERN_MISMATCH=error
ANSIBLE_INVENTORY=/dev/null
ANSIBLE_LIBRARY=/dev/null
ANSIBLE_PLAYBOOK_DIR=/root/ansible/test/results/.tmp/integration/callback_default-HaGZIh-ÅÑŚÌβŁÈ/test/integration/targets/callback_default
...